### ISSUE TITLE
user flow idea: create patient page has only five fields

### ISSUE THREAD

**Issue by mapsam** [codeforamerica/rva-screening#252]
This is an idea based on two things:

Firstly, the "create new patient" workflow is relatively long process. I'm wondering if we are lacking a bit of confirmation for the user that they are doing things correctly along the way (i.e. green successes saying nice things).

Secondly, users have expressed confusion whether or not they need to fill out the entire new patient form right away. They don't but since we provide all of the fields that is their initial feeling.

**I'm proposing turning the "new_patient" page into a small, five field form that includes `first_name`, `middle_name`, `last_name`, `dob`, and `ssn`.** These fields are already going to be partially pre-filled from the index search fields, so this is a quick step. The save button can be a "create patient" and it goes to the patient details page where users can continue filling out information, but this time they have seen a green success notification saying:

```
{{ patient.first_name }} was successfully created! 
Please fill out the rest of their information below in order make accurate referrals.
```

Thoughts @esmithayer and @bengolder?

**Comment by bengolder** [comment]
I totally agree with your points about the process! It seems long, needs user affirmations, and it's rough to confront users with such a big form right away.

I'm wondering if we could go a little further. Could we maybe skip the new_patient form entirely? What if we saved the info that they entered into the search page and then directed them to the patient overview page?

**Comment by mapsam** [comment]
I thought of that as well, and am all for reducing the number of steps. I'm wondering, though, if that starts to over simplify things without giving the user the sense of "constructing this patient" or having the ability to press a "cancel" button when creating a patient. How do you balance simplicity with enough feedback feedback?

**Comment by bengolder** [comment]
Generally I think it's best to make things really simple and allow people to step backwards if needed. Adding cancel along with switching the flow might be just adding too much at this stage. In any case, I think your form proposal is an improvement and we should do it.
